.env.example

```diff
@@ -88,3 +88,6 @@ E2E=true
 
 # Deployment environment
 ENVIRONMENT=
+
+# Feedback form
+FEEDBACK_FORM=
```

Comment:
What's this for @martikat ?

Reply:
I was originally planning to use an env variable to determine whether the feedback form content should be shown (similar to how it is used for disable user answer)